recipes/pymor/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.5.1" %}
+
+package:
+    name: pymor
+    version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/pymor/pymor-{{ version }}.tar.gz
+  sha256: ad5d0e8db09bf65580563632f5a73403790672f8ab33bc943da344fe70d290a0
+
+build:
+  number: 0
+  script:
+    # this is split in two to workaround an opaque msvc error
+    - {{ PYTHON }} setup.py bdist_wheel  # [win]
+    # it's really just one file, but windows shell cannot expand glob patterns
+    - FOR /R %%G in (pymor*.whl) DO {{ PYTHON }} -m pip install %%G --no-deps --ignore-installed -vvv  # [win]
+    # on linux + osx we can do the standard install
+    - {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv  # [not win]
+  skip: true  # [py<35]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+
+  host:
+    - python
+    - pip
+    - cython>=0.20.1
+    - numpy
+    - packaging
+    - pytest-runner
+    - wheel  # [win]
+
+  run:
+    - python
+    - ipython>=3.0
+    - ipyparallel
+    - matplotlib
+    - pyopengl
+    - pyamg  # [not win]
+    - pyqt
+    - pillow
+    - {{ pin_compatible('numpy') }}
+    - scipy>=0.13.3
```

Comment:
There should be a space before the >

Reply:
Fixed the other instances as well, thanks!